server.js

```diff
@@ -177,39 +148,45 @@ function formatTime (time) {
 
 function getUser (userID, amount) {
   return {
-    id: userID, // User id/rethink id
+    userID, // User id/rethink id
```

Comment:
I might be misunderstanding this change, but isn't the `id` required since that's what the database uses behind the scenes?

Reply:
Nice catch, not sure how this got here